.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.15.x]
```

Comment:
go 1.16 is launched and go 1.15 is the latest GAE version. golang-samples tests the oldest and latest supported versions, which in that repo is 1.11 and 1.16. It would be good to do the same to ensure we don't use a language feature that breaks the oldest go version we want to support.

I have not thought about what the oldest supported version should be for this library.

Reply:
I'll use 1.11 and 1.16.